app/Models/Bible/BibleFileset.php

```diff
@@ -464,4 +464,25 @@ public static function getDefaultMetaTags() : array
             BibleFilesetTag::STOCK_NO_TAG => null
         ];
     }
+
+    /**
+     * Filter bible fileset records to avoid pulling the old six character text_plain fileset.
+     *
+     * @param Builder $query
+     */
+    public function scopeConditionToExcludeOldTextFormat(Builder $query) : Builder
+    {
+        $dbp_prod = config('database.connections.dbp.database');
+
+        return $query
+            ->whereNotExists(function (QueryBuilder $subquery) use ($dbp_prod) {
+                return $subquery->select(\DB::raw(1))
+                    ->from($dbp_prod . '.bible_filesets AS bfctext')
+                    ->where('bfctext.set_type_code', BibleFileset::TYPE_TEXT_PLAIN)
+                    ->whereColumn('bfctext.set_type_code', '=', 'bible_filesets.set_type_code')
+                    ->where(DB::raw('CHAR_LENGTH(bible_filesets.id)'), '=', '6')
+                    ->where(DB::raw('CHAR_LENGTH(bfctext.id)'), '>', '6')
```

Comment:
If I am understanding the code correctly, this should really be a test for length 10. And rather than putting a 10 in the query, create a constant NEW_TEXT_PLAIN_FILESET_LENGTH

Reply:
@bradflood , you are right. So, I have added two new constant to store the values related with the length of new and old text plain fileset ID. e.g.

```php
    public const NEW_TEXT_PLAIN_FILESET_LENGTH = 10;
    public const OLD_TEXT_PLAIN_FILESET_LENGTH = 6;
```
Also, I have updated the query to use the above constants. see the following lines:
https://github.com/faithcomesbyhearing/dbp/pull/865/files#diff-c4f12b64765ea311cd0953e83e7e57a88eb1d01ff5ceb4d584a163f5c949e294R487